GW2EIEvtcParser/EncounterLogic/Raids/W7/Sabir.cs

```diff
@@ -177,13 +177,11 @@ internal override void ComputeNPCCombatReplayActors(NPC target, ParsedEvtcLog lo
                         int delay = 3000; // casttime 0 from skill def
                         int duration = 5000;
                         uint radius = 1200;
-                        Point3D targetPosition = target.GetCurrentPosition(log, start + 1000);
-                        if (targetPosition != null)
-                        {
-                            replay.Decorations.Add(new CircleDecoration(radius, (start, start + delay), Colors.Orange, 0.2, new PositionConnector(targetPosition)));
-                            replay.Decorations.Add(new CircleDecoration(radius, (start + delay - 10, start + delay + 100), Colors.Orange, 0.5, new PositionConnector(targetPosition)));
-                            replay.Decorations.Add(new CircleDecoration(radius, (start + delay, start + duration), "rgba(255, 150, 0, 0.7)", new PositionConnector(targetPosition)).UsingFilled(false).UsingGrowingEnd(start + duration));
-                        }
+                        (long, long) lifespanShockwave = (start + delay, start + duration);
+                        GeographicalConnector connector = new AgentConnector(target);
```

Comment:
This was a PositionConnector not AgentConnector, is the change intentional?

Reply:
Yeah, sabir doesn't move during the shockwave so we don't need to check for the position, it wasn't an effect position

Reply:
But the question is: does the shockwave move with Sabir? If not I'd rather keep the old code as that would be the "correct" behavior.

Reply:
We don't know, she will always do the stomp before moving platform and the shockwave will always be over by then, she's quite slow and i've never seen any other behaviour